dotnet/Directory.Packages.props

```diff
@@ -135,7 +135,7 @@
     <PackageVersion Include="Docker.DotNet" Version="3.125.15" />
     <!-- Plugins -->
     <PackageVersion Include="DocumentFormat.OpenXml" Version="3.3.0" />
-    <PackageVersion Include="Microsoft.Data.Sqlite" Version="9.0.0" />
+    <PackageVersion Include="Microsoft.Data.Sqlite" Version="10.0.0-preview.3.25171.6" />
```

Comment:
This bump was required to pickup a bug fix that is required to get it to work (IIRC it was introduced by @krwq in https://github.com/dotnet/efcore/pull/35617)

Reply:
@adamsitnik can you instead try to bump to 9.0.4? IIRC we backported @krwq's fix specifically for this.